README.md

```diff
@@ -18,13 +18,32 @@ $ npm install -g prebuild
 
 ## Building
 
-Create prebuilds for node `v5.4.1` and `0.12.9` (`v` prefix is optional) and write them to `./prebuilds/`
+It's only necessary to build for node/iojs targets with different ABI versions ([Application Binary Interface](https://en.wikipedia.org/wiki/Application_binary_interface)).
+
+To build for *all* known abi versions greater than `0.8`:
+
+```
+prebuild --all
+```
 
+Alternatively, to build for some specific versions you can do:
+
+```
+prebuild -pb 0.10.42 -pb 0.12.10 -pb 4.3.0
 ```
-$ cd a-native-module
-$ prebuild -t v5.4.1 -t 0.12.9
+
+Optionally, to always build for the above versions you can add the following to `~/.prebuildrc`.
+
+``` ini
+prebuild[] = 0.10.42
+prebuild[] = 0.12.10
+prebuild[] = 4.3.0
 ```
 
+Note that `~/.prebuildrc` is a global configuration, i.e. when building *any* native module on your system. If this is not what you want you should consider adding a `.prebuildrc` to your project. This way the module determines which version it supports rather than a global setting.
```

Comment:
We could maybe mention that you can use `~/.prebuildrc` for storing your token, which is a setting you really want globally